python/packages/nisar/pointing/doppler_lut_from_raw.py

```diff
@@ -77,6 +78,10 @@ def doppler_lut_from_raw(raw, *, freq_band='A', txrx_pol=None,
     az_block_dur : float, default=4.0
         Azimuth block duration in seconds defining time-domain correlator
         length used in Doppler estimator.
+    rangeline_limit : tuple[int | None, int | None], optional
+        0-based range line [start, stop] indices to limit echo range lines to
+        be processed. Default is all range lines. The start/stop will be
+        limited to within [0, total rangelines]!
```

Comment:
Let's update this description to make clear that the interval is not inclusive of the right endpoint.

```suggestion
        0-based range line [start, stop) indices to limit echo range lines to
        be processed. Default is all range lines. The start/stop will be
        limited to within [0, total rangelines)!
```

Reply:
(Please make this change in the other scripts as well, thanks!)